assembly/src/parsers/mod.rs

```diff
@@ -171,8 +171,8 @@ impl ProgramAst {
 /// list could be local or exported.
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct ModuleAst {
-    docs: Option<String>,
-    local_procs: Vec<ProcedureAst>,
+    pub docs: Option<String>,
+    pub local_procs: Vec<ProcedureAst>,
 }
```

Comment:
Are these made public so that we can instantiate this struct externally? If so, I'd probably prefer creating a `new()` constructor.

Reply:
Ah, good point. I'll change it.

Reply:
Done.